test/PaletteInsightAgent.UnitTests/ThreadInfo/ThreadInfoAgentTest.cs

```diff
@@ -1,92 +1,53 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
-
+using PaletteInsightAgent.ThreadInfoPoller;
 namespace PaletteInsightAgentTests.ThreadInfo
 {
     /// <summary>
     /// Summary description for ThreadInfoAgentTest
     /// </summary>
     [TestClass]
-    public class ThreadInfoAgentTest
+    public class ThreadInfoAgentTest 
     {
-        public ThreadInfoAgentTest()
+        public void ThreadInfoTest()
         {
-            //
-            // TODO: Add constructor logic here
-            //
         }
 
-        private TestContext testContextInstance;
-
-        /// <summary>
-        ///Gets or sets the test context which provides
-        ///information about and functionality for the current test run.
-        ///</summary>
-        public TestContext TestContext
+        [TestMethod]
+        public void StripProcessNameStartWith()
         {
-            get
-            {
-                return testContextInstance;
-            }
-            set
-            {
-                testContextInstance = value;
-            }
-        }
-
-        #region Additional test attributes
-        //
-        // You can use the following additional attributes as you write your tests:
-        //
-        // Use ClassInitialize to run code before running the first test in the class
-        // [ClassInitialize()]
-        // public static void MyClassInitialize(TestContext testContext)
-        // {
-        // }
-        //
-        // Use ClassCleanup to run code after all tests in a class have run
-        // [ClassCleanup()]
-        // public static void MyClassCleanup() { }
-        //
-        // Use TestInitialize to run code before running each test 
-        [TestInitialize()]
-        public void MyTestInitialize()
+            var processName1 = "control-tabadmincontroller";
+            var processName2 = "run-filestore";
+            var result = ThreadInfoAgent.StripTableauProcessName(processName1);
+            var result2 = ThreadInfoAgent.StripTableauProcessName(processName2);
```

Comment:
I think every test that has `result2` should be separate tests, so I think `StripProcessNameStartWith()` test method could be split into two test methods like: `StripProcessNameStartWithRun()` and `StripProcessNameStartWithControl()`